src/components/EmailsList.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from "react";
+import { ComponentConfig, Fields } from "@measured/puck";
+import {
+  EntityField,
+  YextEntityField,
+  resolveYextEntityField,
+  YextEntityFieldSelector,
+  useDocument,
+} from "@yext/visual-editor";
+import { cn } from "../utils/cn";
+import { cva, type VariantProps } from "class-variance-authority";
+import mailIcon from "../assets/mail_outline.svg";
+
+
+const emailListVariants = cva("list-inside text-font-fontSize p-8", {
+  variants: {
+    fontSize: {
+      xs: "text-xs",
+      sm: "text-sm",
+      medium: "text-base",
+      lg: "text-lg",
+      xl: "text-xl",
+      "2xl": "text-2xl",
+      "3xl": "text-3xl",
+      "4xl": "text-4xl",
+      "5xl": "text-5xl",
+    },
+    fontWeight: {
+      default: "font-body-fontWeight",
+      thin: "font-thin",
+      extralight: "font-extralight",
+      light: "font-light",
+      normal: "font-normal",
+      medium: "font-medium",
+      semibold: "font-semibold",
+      bold: "font-bold",
+      extrabold: "font-extrabold",
+      black: "font-black",
+    },
+    color: {
+      default: "text-body-color",
+      primary: "text-primary",
+      secondary: "text-secondary",
+      accent: "text-accent",
+      text: "text-text",
+      background: "text-background",
+      foreground: "text-foreground",
+    },
+    textTransform: {
+      none: "",
+      uppercase: "ve-uppercase",
+      lowercase: "ve-lowercase",
+      capitalize: "ve-capitalize",
+    },
+    padding: {
```

Comment:
should there be a default that has nothing? then it can go to the theme one automatically

Reply:
I don't think padding was something Aaron has specified for Theme Manager other than for the grid component

Reply:
Removed padding and text transform